src/sagemaker/jumpstart/artifacts/model_packages.py

```diff
@@ -29,6 +29,7 @@
 def _retrieve_model_package_arn(
     model_id: str,
     model_version: str,
+    instance_type: Optional[str],
```

Comment:
Don't we need to do the same for `_retrieve_model_package_model_artifact_s3_uri()`?